m3u8/parser.py

```diff
@@ -325,13 +329,19 @@ def _parse_simple_parameter(line, data, cast_to=str):
     return _parse_and_set_simple_parameter_raw_value(line, data, cast_to, True)
 
 
-def _parse_cueout(line, state):
+def _parse_cueout_cont(line, state):
     param, value = line.split(':', 1)
     res = re.match('.*Duration=(.*),SCTE35=(.*)$', value)
     if res:
         state['current_cue_out_duration'] = res.group(1)
         state['current_cue_out_scte35'] = res.group(2)
 
+def _cueout_no_duration(line):
+    # this needs to be called first since line.split in all other
+    # parsers will throw a ValueError if passed just this tag
+    if line == protocol.ext_x_cue_out:
+        return (None, None)
```

Comment:
Tip: Python does not enforce you to return a tuple. Every return with multiple values will end in a tuple.

Reply:
Good knowledge - I learnt something today! However, if ok, I think I will leave it because it matches the existing style of the other cue parsers?

Reply:
I am ok with it